CHANGELOG.md

```diff
@@ -16,10 +16,14 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 - New method `sdk.alerts.get_all_alert_details()` as a helper to make getting alerts with details easier (combines `sdk.alerts.search_all_pages()` and `sdk.alerts.get_details()`).
 
+- New method `sdk.loginconfig.get_for_user()` to identify if a user's login type is `LOCAL`, `LOCAL_2FA`, or `CLOUD_SSO`.
+
 ### Removed
 
 - py42 no longer supports python 2.7 or python 3.5.
 
+- Usage of `Py42MFARequiredError`. Use `sdk.get_login_configuration_for_user()` instead to check if a user is configured for two-factor authentication.
```

Comment:
This line does not reference the updated method path

Reply:
done